internal/oci/runtime_vm.go

```diff
@@ -243,15 +243,18 @@ func (r *runtimeVM) StartContainer(c *Container) error {
 
 	// Spawn a goroutine waiting for the container to terminate. Once it
 	// happens, the container status is retrieved to be updated.
-	var err error
 	go func() {
-		_, err = r.wait(r.ctx, c.ID(), "")
+		_, err := r.wait(r.ctx, c.ID(), "")
```

Comment:
another nit: personally I prefer multiple returns over if/elses
```
if _, err := r.wait(r.ctx, c.ID(), ""); err != nil {
    logrus.Warnf(...)
    return
}
if err := r.UpdateContainerStatus(c); err != nil {
    logrus.Warnf(...)
}
```
that's really a style nit though, feel free to ignore :smile: